### DISCUSSION TITLE
Vue Echart Tree leaves styling not working properly.

### DISCUSSION THREAD

**Discussion by SyadBabarAli** [npm/feedback#325] [General]
Vue Echart Tree leaves styling not working properly.
![EchartLabelProblem](https://user-images.githubusercontent.com/43900649/114137040-0a43fe80-9925-11eb-9ec5-a222b11ba039.JPG)
Below image leaves text alignment not working when I changed on option object.

**Comment by MylesBorins** [comment]
Hey @SyadBabarAli. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
